client/client.go

```diff
@@ -257,6 +257,7 @@ func (c *Client) processChannelRequests() {
 
 		case *agentpb.StartActionRequest:
 			var action actions.Action
+			actionTimeout := p.GetTimeout()
```

Comment:
If zero, set to 10 seconds (with a comment) – for compatibility with an older server

Reply:
done